src/plugins/chart_expressions/expression_partition_vis/public/utils/layers/get_color.ts

```diff
@@ -145,7 +145,7 @@ const createSeriesLayers = (
   column: Partial<BucketColumns>
 ): SeriesLayer[] => {
   const seriesLayers: SeriesLayer[] = [];
-  let tempParent: typeof arrayNode | typeof arrayNode['parent'] = arrayNode;
+  let tempParent: typeof arrayNode | (typeof arrayNode)['parent'] = arrayNode;
```

Comment:
I see some linting changes in all our files? Are they on purpose? 🤔

Reply:
Hey @stratoula, they actually are not. CI blessed the PR with. I also brought this up with operations the going theory is that some of my modifications triggered the workflow that caused this. I would have preferred the lint changes not to be included in the PR.

Reply:
Interesting, so which change you made triggered these changes? With so many files touched is difficult to understand what is going on

Reply:
I assume it's because the installed version of `prettier` changed? cc @sphilipse

Reply:
I couldn't tell you if I had to guess, I don't actually see anything that looks out for any specific file or directories in the [task definition for linting](https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/lint.sh), so that pretty much leaves the auto_commit_disabled check.

Reply:
Dario is right, it must be the prettier version

Reply:
I can confirm that the lint changes got added in because this PR caused a version change to prettier, see https://github.com/prettier/prettier/pull/14042 for context about why the change did happen and thanks @dgieselaar for the hypothesis!!